apps/wakubridge/wakubridge.nim

```diff
@@ -264,7 +264,7 @@ proc start*(bridge: WakuBridge) {.async.} =
 
   # Always mount relay for bridge.
   # `triggerSelf` is false on a `bridge` to avoid duplicates
-  await bridge.nodev2.mountRelay(triggerSelf = false)
+  await bridge.nodev2.mountRelay()
```

Comment:
Is the `  bridge.nodev2.wakuRelay.triggerSelf = false` missing here? 
￼

Reply:
indeed!

Reply:
done 3d8ba07d5944d5374ccb981c06716db46e727518

Reply:
by now i would say `triggerSelf = true` should be the default value. the bridge is a particular case.